### ISSUE TITLE
[BUG] you might not call it a bug but in the new update they moved some items in the bazaar causing the bazaar flipper to not work properly, and i am currently using the latest version

### ISSUE THREAD

**Issue by H0pebray** [JackTYM/AIOMacro#19]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Please provide the following information:**
 - Your config from `C:\Users\Jackson\AppData\Roaming\.minecraft\config\AIO-Macro\AIO-Macro.toml`
 - Your AIOMacro Version

**Additional context**

**Comment by JackTYM** [comment]
yep, a full bazaar flipper revamp utilizing the search feature is planned to auto this from happening from any future updates
